scripts/ec2/tags.coffee

```diff
@@ -1,10 +1,33 @@
 ec2 = require('../../ec2.coffee')
+moment = require('moment')
 
 
 tags =
-  addSchedule: (msg, instances, schedule = "8:18") ->
+
+  DEFAULT_SCHEDULE: "8:18"
+
+  SCHEDULE_TAG: "RunSchedule"
+
+  # convert our concise schedule strings ("8:18") to the new UTC-based format
+  formatSchedule: (hours) ->
+    [_, start, stop] = hours.match /^(\d{1,2}):(\d{1,2})$/
```

Comment:
I think this expression is checking for an integer number that is either 1 or 2 digits long. If checking is being done here already is there a reason not to do additional checking on whether or not the number can be converted properly to an hour?

For instance what if you were to supply the code here with `"99:88"` ? Could you add an additional check to make sure that numbers were between 0-24? If so, would that be stronger than making sure there are two digits?